src/cpp/src/generation_config.cpp

```diff
@@ -91,6 +91,7 @@ void GenerationConfig::update_generation_config(const ov::AnyMap& config_map) {
     read_anymap_param(config_map, "do_sample", do_sample);
     read_anymap_param(config_map, "repetition_penalty", repetition_penalty);
     read_anymap_param(config_map, "eos_token_id", eos_token_id);
+    read_anymap_param(config_map, "adapters", adapters);
```

Comment:
I suppose generation_config.json does not have `adapters` field

Reply:
It doesn't. This code is used to update the current loaded config with new values provided in properties/property map. In the generation config structure we do have adapters field.

We can try to extend json config with adapters as well, if user wants to hold all parameters in json config file.